contrib/x11/hashblock.h

```diff
@@ -14,53 +13,17 @@
 #include "sph_simd.h"
 #include "sph_echo.h"
 
-#ifndef QT_NO_DEBUG
-#include <string>
-#endif
+#define HASH512_SIZE 64
+#define HASH256_SIZE 32
```

Comment:
This file was taken from
https://github.com/dashevo/dashj-android/tree/master/dashj-x11/src/main/cpp

Which is used to create the native x11 libraries for android.